Portfolio/index.css

```diff
@@ -1,174 +1,126 @@
-console.log('Hello World');
+* {
+    h1 {
+        margin-left: 10px;
+    }
 
-body {
-    font-family: Arial, sans-serif;
-    margin: 0;
-    padding: 0;
-    background-color: #f0f0f0;
 }
-/* Additional CSS styles */
-.exp img{
-
-width: 200px;
-height: 200px;
-border: 2px solid black;
-border-radius: 100px;
 
+.navbar img {
+    margin-left: 10px;
 }
 
-/* Header styling */
-header {
-background-color: #ffffff;
-color: rgb(0, 0, 0);
-text-align: center;
-padding: 20px 0;
-}
+body {
 
-/* Section styling */
-section {
-background-color: #ffffff;
-padding: 20px;
-margin: 20px;

```

Comment:
The opening brace for the '.project-text h1, b' selector is missing on this line.